items/armor/armor.json

```diff
@@ -389,27 +389,7 @@
 		"stamina_regen": 100,
 		"passive": 0
 	},
-	"2090263706": {
-		"name": "CE-07 Demolition Specialist",
```

Comment:
This one seems deleted?

Reply:
This one was mapped to the mixID, fixed to item ID of "1511511766"